c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -2257,6 +2268,38 @@ class TranslateASTVisitor final
 
     bool VisitTypedefNameDecl(TypedefNameDecl *D) {
         auto typeForDecl = D->getUnderlyingType();
+        // If this typedef is to a compiler-builtin macro with target-dependent definition, note the
+        // macro's name so we can map to the appropriate target-independent name (e.g. `size_t`).
+        auto targetDependentMacro = [&]() -> std::optional<std::string> {
+            TypeSourceInfo *TSI = D->getTypeSourceInfo();
+            if (!TSI) {
+                return {};
```

Comment:
Nit: `std::nullopt` is probably a bit clearer here, and for the other `return {}`s.

Reply:
Done,